tests/legacy-cli/e2e/tests/i18n/ivy-localize-es5.ts

```diff
@@ -43,8 +39,6 @@ export default async function() {
   await updateJsonFile('angular.json', workspaceJson => {
     const appProject = workspaceJson.projects['test-project'];
     const appArchitect = appProject.architect || appProject.targets;
-    const serveConfigs = appArchitect['serve'].configurations;
-    const e2eConfigs = appArchitect['e2e'].configurations;
```

Comment:
These can be left in place.  Once `ng serve` support is added, I plan on having both the standalone server and `ng e2e` with serve be run in this test.  The extra configurations will be necessary for `ng serve` since it will only support a single locale.